src/main/app/claims/models/defendantResponse.ts

```diff
@@ -20,11 +25,28 @@ export class DefendantResponse implements Serializable<DefendantResponse> {
       this.id = input.id
       this.claimId = input.claimId
       this.response = new DefendantResponseData().deserialize(input.response)
-      this.defendantDetails = new Defendant().deserialize(input.response.defendant)
+      this.defendantDetails = this.deserializeDefendantDetails(input.response.defendant)
       this.respondedAt = MomentFactory.parse(input.respondedAt)
       this.defendantId = input.defendantId
       this.defendantEmail = input.defendantEmail
     }
     return this
   }
+
+  private deserializeDefendantDetails (defendant: any): Defendant {
+    if (defendant) {
```

Comment:
nothing is returned here if there's no defendant, I assume it works but I would either:
1. Make defendant mandatory
2. return the passed in defendant back
3. return undefined